.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This should be ignored in .gitignore

Reply:
Thank you, I will do that, I didn't notice this.